docs/docs/ref-02-component-api.md

```diff
@@ -16,7 +16,7 @@ Instances of a React Component are created internally in React when rendering. T
 ```javascript
 setState(function|object nextState[, function callback])
 ```
-Merges nextState with the current state. This is the primary method you use to trigger UI updates from event handlers and server request callbacks.
+Merges (shallow merge) nextState with the current state. This is the primary method you use to trigger UI updates from event handlers and server request callbacks.
```

Comment:
I feel like this reads a little awkwardly.  Maybe "Merges (shallowly, not recursively) nextState with the current state" or "Merges nextState with the current state (shallowly, not recursively)".  Or reword so it flows more naturally with the paragraph text.  The phrase "Merges (shallow Merge) nextState" just feels awkward; the rewording has the added bonus of clearifying what shallow means for anyone unfamiliar with the term (intro-to-react api).  That said, I don't have a strong objection here.  @zpao or @spicyj for a final decision here.